sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldValue_internal.cs

```diff
@@ -8,6 +8,9 @@ namespace Azure.AI.FormRecognizer.Models
     [CodeGenModel("FieldValue")]
     internal partial class FieldValue_internal
     {
+        /// <summary>Integer value.</summary>
+        public long? ValueInteger { get; }
```

Comment:
Explicitly declaring `ValueInteger` as a `long?` so CodeGen can generate it with the correct type (instead of `int?`, which is the current default for integer).